Portal/vip-portal/pom.xml

```diff
@@ -148,6 +162,14 @@ knowledge of the CeCILL-B license and that you accept its terms.
                 <skipModule>true</skipModule>
               </configuration>
             </plugin>
+
+            <plugin>
+              <artifactId>maven-war-plugin</artifactId>
+              <version>3.2.3</version>
+              <configuration>
+                <packagingExcludes>WEB-INF/lib/*-sources.jar</packagingExcludes>
+              </configuration>
+            </plugin>
```

Comment:
Is it really necessary ? I thought that `<scope>provided</scope>` would have make the source jar absent from the war.

Reply:
Yes, tests have shown that this is necessary, to exclude source jars from the war.  And they are 2 sources jar from libraries used that are also excluded with this configuration.